Employee.java

```diff
@@ -0,0 +1,77 @@
+package extraction.sProject;
+import java.util.*;
+public class Employee {
+	
+	private String employeeID;
+	private String employeeName;
+	private String employeeEmail;
+	private double classID;
+	private List<Module> modScores = new ArrayList<>();	
```

Comment:
Does this work? Shouldn't you have the same class name on both sides? Like... If you are going to use an ArrayList, which I recommend then declare the var as such.